earn/src/pages/InfoPage.tsx

```diff
@@ -215,9 +216,29 @@ export default function InfoPage() {
         }
       });
 
+      // Get the time at which each pool was last updated via the oracle (using the update event and getLogs)
+      const latestTimestamps = await Promise.all(
+        poolAddresses.map(async (addr) => {
+          let updateLogs: ethers.providers.Log[] = [];
+          try {
+            updateLogs = await provider.getLogs({
+              address: ALOE_II_ORACLE_ADDRESS[chainId],
+              topics: [TOPIC0_UPDATE_ORACLE, null, `0x000000000000000000000000${addr.slice(2)}`],
+              fromBlock: 0,
+              toBlock: 'latest',
+            });
+            if (updateLogs.length > 0) {
+              const latestUpdate = updateLogs[updateLogs.length - 1].blockNumber;
```

Comment:
Probably want to filter for logs where `removed === false`

See return value docs here: https://www.quicknode.com/docs/ethereum/eth_getLogs